Box.V2/Config/Constants.cs

```diff
@@ -880,5 +883,19 @@ public static class WhitelistDirections
             /// </summary>
             public const string Both = "both";
         }
+
+        /*** The desired conflict-resolution if a template already exists on a given file or folder.***/
+        public static class ConflictResolution
```

Comment:
How would a user access this?  Can we make some of the tests/documentation reference this?